TimberWinR/mdocs/Generator.md

```diff
@@ -0,0 +1,37 @@
+# Input: Generator
+
+The Generator input can be used to Generate log files for test purposes.
+
+## Parameters
+The following parameters are allowed when configuring WindowsEvents.
```

Comment:
Generator is a convience for generating logs for testing, same as logstash:

http://logstash.net/docs/1.4.2/inputs/generator